packages/zarathustra/skills/asylum_abci_app/scraper.py

```diff
@@ -50,8 +49,7 @@ def save_user_data(self, username: str, issues: dict) -> str | None:
         if not user_dir:
             return None
 
-        timestamp = datetime.now(UTC).strftime("%Y%m%d_%H%M%S")
-        filename = f"data_{timestamp}.json"
+        filename = "repos.json"
```

Comment:
deterministic. Should be defined here (at least the pattern) and then imported into behaviour to check existence, now redundantly specified there still